src/geff/metadata_schema.py

```diff
@@ -140,6 +184,10 @@ class GeffMetadata(BaseModel):
     )
     directed: bool
     axes: Sequence[Axis] | None = None
+    related_objects: Sequence[RelatedObject] | None = Field(
+        None,
+        description=("A list of dictionaries of related objects such as labels or images."),
```

Comment:
This description should include a definition of the `RelatedObject` type, because the docstrings of the `RelatedObject` won't get pulled into the schema.